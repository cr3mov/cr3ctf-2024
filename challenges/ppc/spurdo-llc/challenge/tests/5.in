100 187348
635385 635385 484929 635385 635385 635385 635385 635385 635385 635385 175317 635385 635385 635385 635385 635385 635385 635385 635385 15576 635385 635385 155928 635385 635385 635385 635385 635385 323043 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 113401 635385 405180 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 36953 635385 635385 635385 635385 23144 635385 635385 635385 635385 635385 635385 635385 635385 635385 6390 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385 635385
