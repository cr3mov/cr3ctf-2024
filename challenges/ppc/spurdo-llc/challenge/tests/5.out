633303 633303 484929 633303 633303 633303 633303 633303 633303 633303 175317 633303 633303 633303 633303 633303 633303 633303 633303 15576 633303 633303 155928 633303 633303 633303 633303 633303 323043 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 113401 633303 405180 633303 633303 633303 633303 633303 633303 633303 633303 633303 633303 633304 633304 633304 633304 633304 633304 633304 633304 36953 633304 633304 633304 633304 23144 633304 633304 633304 633304 633304 633304 633304 633304 633304 6390 633304 633304 633304 633304 633304 633304 633304 633304 633304 633304 633304 